knocking the cover (Delegate discretion) (#31)
CarterKoala/wca-regulations-wrc: wca-regulations.md

        - A2d1) The scrambler or an authorized judge must check the scrambled puzzle, and sign (or initial) the score sheet to affirm that they applied the correctly-numbered scramble sequence and checked that it matches the image (see [Regulation 4g](regulations:regulation:4g)). Exception: for 6x6x6 Cube, 7x7x7 Cube, and Megaminx the scrambler may sign (or initial) to affirm that the puzzle is sufficiently scrambled (see [Regulation 4g1](regulations:regulation:4g1)).
    - A2e) The judge places the puzzle onto the mat in an arbitrary orientation while ensuring that it remains completely covered.
        - A2e1) The competitor is not permitted to request a specific orientation, and the judge must not influence the orientation they place the puzzle on the mat based on what they know or expect the puzzle state to be.

>         - A2e2) If the competitor accidently knocks the cover off and the puzzle is partially or fully visible to the competitor, an extra may be provided instead of disqualification of the attempt (DNF), at the discretion of the WCA Delegate.

Reviewer: Option for DNFing this every time:

```suggestion
        - A2e2) The competitor must not accidentally or intentionally remove the cover themselves. Penalty: disqualification of the attempt (DNF).
```

Reviewer 2: ```suggestion
        - A2e2) The competitor must not remove the cover themselves. Penalty: disqualification of the attempt (DNF).
```
Is this more elegant? We don't have to bring intentionality into the regs unless it makes a difference, which it doesn't here.

Reviewer 3: I like Hex's suggestion.

>         - A2e2) If the compeititor accidently knocked the cover off and the puzzle is partially or fully visible to the competiitor, an extra may be provided instead of disqualification of the attempt (DNF), at the discretion of the WCA Delegate.

Reviewer 2: ```suggestion
        - A2e2) If the competitor accidently knocks the cover off and the puzzle is partially or fully visible to the competitor, an extra may be provided instead of disqualification of the attempt (DNF), at the discretion of the WCA Delegate.
```
Spelling corrections, align tense with other regs
